### ISSUE TITLE
Nav Bar

### ISSUE THREAD

**Issue by dan9d9** [chingu-voyages/v11-toucans-team-13#17]

**Comment by dan9d9** [comment]
All of the nav buttons open up new windows, none of them scroll to a new part of the page. So we don't have to worry about smooth scrolling.

Do we want to include the dropdown "Tap.Eat.Learn" above the nav list? Or do we focus first on the rest of the page, and then add it if we have time? I don't think it's essential for our MVP. I think we can add it as a bonus issue later if we have time.

**Comment by MinervaLong** [comment]
I agree with you, I prefer to focus in the drop-down menu for example, instead of the tap. If we have time we can add it later
